Workflow/rebuild_cache

```diff
@@ -35,11 +36,17 @@ db = SQLite3::Database.new(Tmp_file.to_path)
 db.execute('CREATE TABLE main (fullpath TEXT, basename TINYTEXT, isdir BOOLEAN, accesstime INTEGER);')
 
 Gdrive_paths.each do |gdrive_path|
-  gdrive_path.glob('**/*').each do |path|
+  # Use system `find` utility instead of Ruby glob for stability and speed
+  cmd = "find '#{gdrive_path.to_path}' -mindepth 1 > '#{Paths_file.to_path}'"
+  cmd_result = system( cmd )
+  abort 'Path traversal did not succeed' if not cmd_result
+  
+  File.open(Paths_file.to_path, 'r').each do |line|
+    path = Pathname.new(line)
     next if path.symlink?
     next if Ignores.any? { |i| path.include?(i) }
 
-    accesstime = path.atime.to_i rescue next
+    accesstime = path.atime.to_i rescue Time.new(1970).to_i
```

Comment:
Only `0` is needed, because we’re referring to the beginning of the UNIX epoch and `Time` is being converted to an integer anyway.